tapdb/assets_store.go

```diff
@@ -726,7 +764,12 @@ func dbAssetsToChainAssets(dbAssets []ConfirmedAsset,
 // constraintsToDbFilter maps application level constraints to the set of
 // filters we use in the SQL queries.
 func constraintsToDbFilter(query *AssetQueryFilters) QueryAssetFilters {
-	var assetFilter QueryAssetFilters
+	assetFilter := QueryAssetFilters{
+		Now: sql.NullTime{
+			Time:  time.Now().UTC(),
```

Comment:
Not blocking at all here, but I think we'll want to eventually start to use `clock.Clock` (from lnd) here so we can eventually mock out the time for unit tests.